Update mainnet_rules.md

Asgaror opened the pull request:
Add new board ROCK Pi S by Radxa to accepted list https://wiki.radxa.com/RockpiS

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/skycoin/skywire?pullRequest=592) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/skycoin/skywire?pullRequest=592) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/skycoin/skywire?pullRequest=592) it.</sub>

Asgaror committed:
Update mainnet_rules.md

Add new board ROCK Pi S by Radxa to accepted list https://wiki.radxa.com/RockpiS
